hoss/bbox_utilities.py

```diff
@@ -58,6 +58,7 @@ def get_request_shape_file(message: Message, working_dir: str,
             raise UnsupportedShapeFileFormat(message.subset.shape.type)
 
         shape_file_url = message.subset.shape.process('href')
+        adapter_logger.info('Downloading request shape file')
```

Comment:
Initially I included the path to the shape file, but this is somewhat meaningless as Harmony creates a temporary file, which obfuscates any original user naming convention (the file name is essentially a generated ID).